NET Core/LibUA/NetDispatcher.cs

```diff
@@ -559,19 +559,16 @@ protected override int Consume(MemoryBuffer recvBuf)
                         return DispatchHello(config, recvBuf);
                     }
 
-                    logger?.Log(LogLevel.Error, string.Format("{0}: TL already set", LoggerID()));
+                    logger?.Log(LogLevel.Error, "{LoggerID}: TL already set", LoggerID());
 
                     return ErrorInternal;
                 }
                 else if (messageType == (uint)MessageType.Open)
                 {
                     if (config.TL == null)
                     {
-                        if (logger != null)
```

Comment:
@nauful do you see any problem with actually returning the ErrorInternal here?
Previous behavior was to just ignore this error (in this location) if the logger didn't exist.
There's a few such instances similar to this.

If this is behavior that is desired, perhaps we could put it behind a boolean flag for 'HideInternalErrors' (defaulting to false) instead (to decouple it from the presence/absence of a logger).